frontend/src/components/notification/notification-dropdown.tsx

```diff
@@ -75,9 +78,9 @@ const NotificationDropdown = ({user, token}: UserProps) => {
             ): ""}
             <DropdownMenu>
                 <DropdownMenuTrigger asChild>
-                    <Button variant="ghost">
-                        <BellIcon className="h-4 w-4 mr-2" />
-                        Notifications
+                    <Button variant="ghost" className="relative">
+                        <Bell className="h-4 w-4 lg:mr-2" />
+                        <p className="hidden xl:block">Notifications</p>
```

Comment:
Move user notifications here for spacing
```suggestion
                        <p className="hidden xl:block">Notifications {userNotifications.length ? `(${userNotifications.length})` : ""}</p>
```